src/com/zone5cloud/retrofit/core/OkHttpClientInterceptor_Authorization.java

```diff
@@ -249,6 +263,7 @@ private String saveNewToken(String path, Response response) {
 		        				login.setTokenExp(System.currentTimeMillis() + (login.getExpiresIn() * 1000));
 		        			}
 		        			setToken(new OAuthToken(login));
+							setUserName(login.getUser().getEmail());
 		        			body = GsonManager.getInstance().toJson(login, Types.LOGIN_RESPONSE);
```

Comment:
do null check on getUser() and getEmail(). This is already inside a login != null block, so just need:
```
if (login.getUser() != null && login.getUser().getEmail() != null) {
    setUserName(login.getUser().getEmail());
}
```